core/opendaq/device/include/opendaq/device.h

```diff
@@ -237,7 +237,7 @@ DECLARE_OPENDAQ_INTERFACE(IDevice, IFolder)
      * @brief Loads the configuration of the device from string.
      * @param configuration Serialized configuration of the device.
      */
-    virtual ErrCode INTERFACE_FUNC loadConfiguration(IString* configuration) = 0;
+    virtual ErrCode INTERFACE_FUNC loadConfiguration(IString* configuration, IPropertyObject* config = nullptr) = 0;
```

Comment:
I'd like to see a special interface that inherits IPropretyObject here so that users know what settings are available.